package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "scc-broker",
-  "version": "1.6.4",
+  "version": "1.6.6",
```

Comment:
Like scc-state PR, no need to bump the version number between each commit; just once at the end ;p
Also I only bump up when I'm about to do a release so you could just let me do it.

Reply:
Sorry for the version bumps. I won't do them anymore. Apparently I have `npm version patch` somewhere beneath my `git add --force commit --force push --force npm publish --force` goddamn shell alias :joy_cat: